src/v/datalake/translation/partition_translator.cc

```diff
@@ -182,9 +205,7 @@ partition_translator::do_translation_for_range(
     auto writer_factory = std::make_unique<local_parquet_file_writer_factory>(
       local_path{_writer_scratch_space}, // storage temp files are written to
       fmt::format("{}", begin_offset),   // file prefix
-      ss::make_shared<batching_parquet_writer_factory>(
-        max_rows_per_row_group,    // max entries per single parquet row group
-        max_bytes_per_row_group)); // max bytes per single parquet row group
+      get_parquet_writer_factory());
```

Comment:
are there tests that don't yet work with the new writer?

Reply:
no, all the tests we have works with both new and old writers